src/routes/userRouter.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const express = require('express');
```

Comment:
Consider adding a newline after the 'use strict' statement for better readability.